src/main/java/fr/openmc/core/features/contest/menu/TradeMenu.java

```diff
@@ -59,7 +59,10 @@ public TradeMenu(Player owner) {
 
         String campName = contestPlayerManager.getPlayerCampName(player);
         NamedTextColor campColor = contestManager.dataPlayer.get(player.getUniqueId().toString()).getColor();
-        Material shell_contest = CustomStack.getInstance("contest:contest_shell").getItemStack().getType();
+
+        // ITEM ADDER
+        String namespaceShellContest = "omc_contest:contest_shell";
+        Material shell_contest = CustomItemRegistry.getByName(namespaceShellContest).getBest().getType();
```

Comment:
Ce nom de variable ne respecte pas les règles de contribution.
https://github.com/ServerOpenMC/PluginV2/blob/master/CONTRIBUTING.md#-style-de-code
```suggestion
        Material shellContest = CustomItemRegistry.getByName(namespaceShellContest).getBest().getType();
```